src/dmd/dmangle.d

```diff
@@ -392,6 +392,8 @@ public:
             buf.writestring("Nd");
         if (ta.isnogc)
             buf.writestring("Ni");
+        if (ta.islive)
+            buf.writestring("??");
```

Comment:
NEEDS FIX!

Reply:
Since all function attributes are mangled as `N<small-letter>, according to: https://dlang.org/spec/abi.html#FuncAttrPure, I suggest:

```suggestion
            buf.writestring("Nk");
```

Reply:
I found this in core.demangle:

> The return parameter type is represented as "Nk"
https://github.com/dlang/druntime/commit/87fbb2885462f6a869161aa581efa21231d693d3

Reply:
You're right, thanks for finding that!